docs/content/build.md

```diff
@@ -58,3 +58,14 @@ This starts the build process for the worker and deployment type (cloud or local
 > **It’s finished building when:** there is a message stating `Completed build for Local/Cloud target`.
 
 You can now find the built-out workers in `build/assembly/worker` in the root of the SpatialOS GDK for Unity project.
+
+#### TIP: Fast worker build with quick-run
```

Comment:
this tip could probably be renamed to something like "Iterate faster locally with quick-run"

Reply:
Ta - have had a rejig